### PR TITLE
Add Service cluster cidr and kubedns ip and san value for cert

### PR DESCRIPTION
**Pull request by galal-hussein**
fix for https://github.com/rancher/rancher/issues/9310

Depends on https://github.com/rancher/kubernetes-package/pull/92

### CODE REVIEW

File: infra-templates/k8s/34/rancher-compose.yml [hunk 1]
```diff
@@ -67,6 +67,24 @@ catalog:
...
+    type: string
+  - variable: DNS_CLUSTER_IP
+    label: KubeDNS cluster IP
+    description: The IP address assigned to kubedns service. This must be within the range of Service Cluster IP CIDR.
+    required: true
+    default: "10.43.0.10"
+    type: string
+  - variable: KUBEAPI_CLUSTER_IP
+    label: KubeAPI cluster IP for TLS certificate
+    description: This is the IP address that will be added as a SAN to the TLS certificate for kubernetes api, this must matches the IP of the kubernetes master service.
```
Comment (alena1108) [inline infra-templates/k8s/34/rancher-compose.yml]: Similar to dns, we should mention that the ip address must be the first ip of the Service Cluster IP CIDR

File: infra-templates/k8s/34/rancher-compose.yml [hunk 2]
```diff
@@ -67,6 +67,24 @@ catalog:
     required: false
     default: false
     type: boolean
+  - variable: SERVICE_CLUSTER_CIDR
+    label: Service Cluster IP CIDR
+    description: A CIDR notation IP range from which to assign service cluster IPs. This must not overlap with any IP ranges assigned to ipsec cidr (10.42.0.0/16).
```
Comment (alena1108) [inline infra-templates/k8s/34/rancher-compose.yml]: We should refer to Rancher ipsec cidr in a more generic manner given that its value is configurable as well.

File: infra-templates/k8s/34/rancher-compose.yml (hunk 1 above)
Comment (galal-hussein) [reply infra-templates/k8s/34/rancher-compose.yml]: got it, will add this part

File: infra-templates/k8s/34/rancher-compose.yml (hunk 2 above)
Comment (galal-hussein) [reply infra-templates/k8s/34/rancher-compose.yml]: in the Ipsec catalog the cidr its just called subnet, its description is:

`The subnet to use for the managed IPSEC network.`

i will change it to `This must not overlap with any IP ranges assigned to the cluster pod cidr (10.42.0.0/16)`

Comment (alena1108) [reply infra-templates/k8s/34/rancher-compose.yml]: You can also mention the default value of the subnet.

File: infra-templates/k8s/33/README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
 ### Upgrading to this Version
 
 Warning: The existing template version _must be_ `v1.2.4-rancher9` or later. Ignoring this will result in data loss. For older templates, please first upgrade to `v1.5.4-rancher1`.
 
-### Changelog for Kubernetes 1.7.4
+### Changelog for Kubernetes 1.8.0-beta
 
-* Added healthcheck to addon-starter
+* Added service cluster ip cidr
```
Comment (alena1108) [inline infra-templates/k8s/33/README.md]: You think it is better to say that we made it configurable?

**Comment by alena1108** [comment]
LGTM

File: infra-templates/k8s/33/rancher-compose.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 .catalog:
   name: Kubernetes
-  version: v1.7.4-rancher2-etcdv2
+  version: v1.8.0-beta.1-rancher2-etcdv2
   description: Rancher Kubernetes service
   minimum_rancher_version: v1.6.9-rc1
```
Comment (deniseschannon) [inline infra-templates/k8s/33/rancher-compose.yml]: Can we update these to `v1.6.10-rc1` and `v1.6.10`?

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**